frontend/utilities/helpers.tsx

```diff
@@ -53,6 +53,31 @@ import { IScheduledQueryStats } from "interfaces/scheduled_query_stats";
 const ORG_INFO_ATTRS = ["org_name", "org_logo_url"];
 const ADMIN_ATTRS = ["email", "name", "password", "password_confirmation"];
 
+/**
+ *
+ * @param count The number of items.
+ * @param root The root word, omitting any suffixs.
+ * @param pluralSuffix The suffix to add to the root word if the count is not 1. Defaults to "s".
+ * @param singularSuffix The suffix to add to the root word if the count is 1. Defaults to "".
+ * @returns A string with the root word and the appropriate suffix.
+ *
+ * @example
+ * pluralize(1, "hero", "es", "") // "hero"
+ * pluralize(0, "hero", "es", "") // "heroes"
+ * pluralize(1, "fair", "ies", "y") // "fairy"
+ * pluralize(2, "fair", "ies", "y") // "fairies"
+ * pluralize(1, "dragon") // "dragon"
+ * pluralize(2, "dragon") // "dragons"
+ */
+export const pluralize = (
+  count: number,
+  root: string,
+  pluralSuffix = "s",
+  singularSuffix = ""
+) => {
+  return `${root}${count !== 1 ? pluralSuffix : singularSuffix}`;
```

Comment:
this is nice but can quickly get quite complicated from my experience. I think for now this is fine but would want to lean on a 3rd party library for this functionality going forward.

Reply:
Yeah, I agree, this is not an approach that can be applied wholesale. Since a lot of the third-party libraries are heavier, dictionary-based approaches in order to be comprehensive so I just wanted to go with a very simple, lightweight approach and empower the user to choose whether they need a more complex solution for their particular case. With that in mind, I think I'll remove the default values so the user is forced to think about it by TypeScript.